### ISSUE TITLE
Redo release schedule page once/during 4.0 release

### ISSUE THREAD

**Issue by ShyamsundarR** [gluster/glusterweb#170]
In this page: https://www.gluster.org/release-schedule/ needs some restructuring.

1. Remove pre-3.8 release schedule details around 4.0 release (this content is old enough to be removed now)
2. Explain current release schedule a little better (a lot of it is captured, but needs some massaging post (1) )
3. Header jumps into explaining minor releases, which should not be how the page starts
4. Also the same content is repeated near the table below

So in short reconsider this page and rehash it to read better.

**Comment by amye** [comment]
How about we do this: 
- Start with what the most current release is, link to the feature page and blog post. This is the new sidebar next to the release schedule. 
- Remove the chart? I don't think it helps us anymore. 
- Drop the minor releases to the bottom of the page. 
- Profit?

**Comment by amye** [comment]
Fixed. Happy to have more review on this if we want to change this moving forward.
